evm_rpc_types/src/rpc_client/mod.rs

```diff
@@ -1,15 +1,34 @@
 pub use ic_cdk::api::management_canister::http_request::HttpHeader;
+use std::fmt::Debug;
 
 use candid::{CandidType, Deserialize};
 use serde::Serialize;
+use strum::VariantArray;
 
 #[derive(Clone, Debug, PartialEq, Eq, Default, CandidType, Deserialize)]
 pub struct RpcConfig {
     #[serde(rename = "responseSizeEstimate")]
     pub response_size_estimate: Option<u64>,
+
+    #[serde(rename = "responseConsensus")]
+    pub response_consensus: Option<ConsensusStrategy>,
+}
+
+#[derive(Clone, Debug, PartialEq, Eq, Default, CandidType, Deserialize)]
+pub enum ConsensusStrategy {
+    /// All providers must return the same non-error result.
+    #[default]
+    Equality,
+    Threshold {
+        /// Number of providers to be queried.
+        /// Can be omitted, if caller specifies manually the providers to be used.
```

Comment:
```suggestion
        /// Can be omitted if the caller specifies the providers to be used manually.
```

Reply:
Should we specify here what `num_providers: None` means?

Reply:
I'm not sure what you mean, this is what the sentence you commented on is about. Do you have maybe a concrete suggestion?

Reply:
I gave a bit more details in this [commit](https://github.com/internet-computer-protocol/evm-rpc-canister/pull/286/commits/4bd0139c1f01567dcf598a19c4ce00b1cc71df53), PTAL.